I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 67, in script call
    call get_studies
  File "game/patterns.rpy", line 11, in script call
    call mini_games
  File "game/mini games.rpy", line 19, in script call
    call mini_game_9
  File "game/mini games.rpy", line 328, in script
    p '''
IndexError: tuple index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 67, in script call
    call get_studies
  File "game/patterns.rpy", line 11, in script call
    call mini_games
  File "game/mini games.rpy", line 19, in script call
    call mini_game_9
  File "game/mini games.rpy", line 328, in script
    p '''
  File "D:\renpy-8.0.3-sdk\renpy\ast.py", line 921, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "D:\renpy-8.0.3-sdk\renpy\exports.py", line 1373, in say
    who(what, *args, **kwargs)
  File "D:\renpy-8.0.3-sdk\renpy\character.py", line 1253, in __call__
    what = self.prefix_suffix("what", self.what_prefix, what, self.what_suffix)
  File "D:\renpy-8.0.3-sdk\renpy\character.py", line 1172, in prefix_suffix
    return (sub(prefix) + sub(body) + sub(suffix))
  File "D:\renpy-8.0.3-sdk\renpy\character.py", line 1152, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "D:\renpy-8.0.3-sdk\renpy\substitutions.py", line 278, in substitute
    s = formatter.vformat(s, (), kwargs) # type: ignore
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 165, in vformat
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 205, in _vformat
  File "D:\renpy-8.0.3-sdk\renpy\substitutions.py", line 168, in get_field
    obj, arg_used = super(Formatter, self).get_field(field_name, args, kwargs)
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 270, in get_field
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/string.py", line 225, in get_value
IndexError: tuple index out of range

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
Student Simulator 1.0
Fri May 12 21:25:15 2023
